perf/_runner.py

```diff
@@ -666,6 +676,36 @@ def _spawn_worker(self, calibrate=False):
 
         return _load_suite_from_pipe(bench_json)
 
+    def _spawn_worker_windows(self, calibrate=False):
+        import tempfile
+        wpipe, tmp_file = tempfile.mkstemp()
+        os.close(wpipe)
+        os.remove(tmp_file)  # remove for subprocess to create
```

Comment:
It seems overkill to create a file just to remove it: please use tempfile.mktemp(). By the way, perf/tests/__init__.py already contains such function. I suggest to move temporary_file() to perf/_utils.py and uses it here.